src/App.tsx

```diff
@@ -15,49 +18,55 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+interface PrepearedGoods {
+  sortField: string;
```

Comment:
please fix the typing for `sortField` everywhere
```suggestion
  sortField: SortType;
```